src/Model_Hydro/CPU_Hydro/CPU_Shared_DataReconstruction.cpp

```diff
@@ -625,6 +670,14 @@ void Hydro_DataReconstruction( const real g_ConVar   [][ CUBE(FLU_NXT) ],
    __syncthreads();
 #  endif
 
+
+#  if ( FLU_SCHEME == MHM && defined MHD )
+// 9. Store the half-step primitive variables for MHM+MHD
```

Comment:
Emphasize that it must be called after the `CGPU_LOOP( idx_fc, CUBE(N_FC_VAR) )` loop since this function will update `g_PriVar[]`.